lib/diagnostic.rb

```diff
@@ -78,10 +98,12 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = 'I am Animal and I go "meow"'
 
 ## Explain why this would be the output, based on the method lookup chain.
-# your response as a comment here
+'sound would look to the current class whereas self.class.name would look
```

Comment:
`self.class.name` would be evaluated to `HouseCat.class.name`, which would be evaluated to `HouseCat.name`, which would be evaluated to `"HouseCat"`, since `say_hello` is a superclass method being called from `HouseCat`.